### PR TITLE
Log Context: Improved "Whats New" text

### PR DESCRIPTION
**Pull request by svennergr**
Moved the Log Context part to the `Dashboards and visualizations` and improved the text.

### CODE REVIEW

File: docs/sources/whatsnew/whats-new-in-v10-0.md [hunk 1]
```diff
@@ -198,6 +198,21 @@ We’ve improved the way you can configure annotations by adding the possibility
...
 
+### Redesigned and improved Log Context
+
+Our team has recently made enhancements to Grafana's log context feature, resulting in a more seamless and consistent user experience. With the updated user interface, users can expect the same level of functionality and usability in log context as they would in any other logs panel.
+
+Notably, we've added the follwoing new features that streamline the log context experience:
+
+1. Log details with actions including a "copy" button for easily copying lines and an eye icon for displaying only selected labels, allowing users to focus on specific information without leaving the log context panel.
+2. A "Wrap Lines" toggle to automatically wrap long lines of text for easier reading and analysis of log entry context directly in log context.
+3. To execute the context query for a log entry in a split screen in the Explore view we've added a "Open in split view" button.
```
Comment (matyax) [inline docs/sources/whatsnew/whats-new-in-v10-0.md:209]: Given the way the enumeration is described ("the following new  features"), maybe:

```suggestion
3. An "Open in split view" button to execute the context query for a log entry in a split screen in the Explore view.
```

**Review by matyax — APPROVED** [review]

File: docs/sources/whatsnew/whats-new-in-v10-0.md [hunk 2]
```diff
@@ -198,6 +198,21 @@ We’ve improved the way you can configure annotations by adding the possibility
 
 {{< figure src="/media/docs/grafana/screenshot-grafana-10-0-annotation-filtering.png" max-width="750px" caption="Annotation filtering" >}}
 
+### Redesigned and improved Log Context
+
+Our team has recently made enhancements to Grafana's log context feature, resulting in a more seamless and consistent user experience. With the updated user interface, users can expect the same level of functionality and usability in log context as they would in any other logs panel.
+
+Notably, we've added the following new features that streamline the log context experience:
+
+1. Log details with actions including a "copy" button for easily copying lines and an eye icon for displaying only selected labels, allowing users to focus on specific information without leaving the log context panel.
```
Comment (JStickler) [inline docs/sources/whatsnew/whats-new-in-v10-0.md:207]:
```suggestion
1. Log details with actions including a **Copy** button to easily copy lines and an eye icon to display only selected labels, allowing users to focus on specific information without leaving the log context panel.
```

File: docs/sources/whatsnew/whats-new-in-v10-0.md [hunk 3]
```diff
@@ -198,6 +198,21 @@ We’ve improved the way you can configure annotations by adding the possibility
...
+2. A "Wrap Lines" toggle to automatically wrap long lines of text for easier reading and analysis of log entry context directly in log context.
```
Comment (JStickler) [inline docs/sources/whatsnew/whats-new-in-v10-0.md:208]:
```suggestion
2. A **Wrap Lines** toggle to automatically wrap long lines of text for easier reading and analysis of log entry context directly in log context.
```

File: docs/sources/whatsnew/whats-new-in-v10-0.md [hunk 4]
```diff
@@ -198,6 +198,21 @@ We’ve improved the way you can configure annotations by adding the possibility
...
+3. An "Open in split view" button to execute the context query for a log entry in a split screen in the Explore view.
```
Comment (JStickler) [inline docs/sources/whatsnew/whats-new-in-v10-0.md:209]:
```suggestion
3. An **Open in split view** button to execute the context query for a log entry in a split screen in the Explore view.
```

File: docs/sources/whatsnew/whats-new-in-v10-0.md [hunk 5]
```diff
@@ -198,6 +198,21 @@ We’ve improved the way you can configure annotations by adding the possibility
...
+4. Only for Loki: A quick filter menu that let's you easily refine the context query by selecting and removing labels.
```
Comment (JStickler) [inline docs/sources/whatsnew/whats-new-in-v10-0.md:210]:
```suggestion
4. Only for Loki: A quick filter menu that lets you easily refine the context query by selecting and removing labels.
```

**Review by JStickler — COMMENTED** [review]
[Docs team] Use bold for UI elements like buttons.

**Review by JStickler — APPROVED** [review]
[docs team] LGTM

**Pull request merged**